src/screens/SplashScreen/index.js

```diff
@@ -5,7 +5,8 @@ import {Linking} from 'react-native';
 import {StackActions} from '@react-navigation/native';
 import {useNavigation} from '@react-navigation/core';
 
-import useProfileHook from '../../hooks/core/profile/useProfileHook';
+import useUserAuthHook from '../../hooks/core/auth/useUserAuthHook';
+import TokenStorage, {ITokenEnum} from '../../utils/storage/custom/tokenStorage';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useUserAuthHook` and `TokenStorage` seems to be correct. The new hook `useUserAuthHook` is replacing the old `useProfileHook`, and `TokenStorage` is being used to retrieve tokens.
```diff
- import useProfileHook from '../../hooks/core/profile/useProfileHook';
+ import useUserAuthHook from '../../hooks/core/auth/useUserAuthHook';
+ import TokenStorage, {ITokenEnum} from '../../utils/storage/custom/tokenStorage';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->